docs/docs/outputs/output-formats.md

```diff
@@ -1,78 +1,60 @@
 # Tracing Output Formats
 
-<!-- TODO: build man page -->
-The `--output` flag controls where and how Tracee will output events, by specifying `--output <format>:<destination>`.  You can use the `--output` flag multiple times to output events in multiple ways.<!-- To see all output options you can run `man tracee-output`. -->
+It is possible to define how the events that Tracee collects should be displayed. This is done through the Tracee configuration. You can read more on configuration in the [Tracee installation guide.](../install/index.md)
 
-The following output formats are supported:
-
-- `table[:/path/to/file]` - output events in table format (default). The default path to file is stdout.
-- `table-verbose[:/path/to/file]` - output events in table format with extra fields per event. The default path to file is stdout.
-- `json[:/path/to/file]` - output events in json format. The default path to file is stdout.
-- `gob[:/path/to/file]` - output events in gob format. The default path to file is stdout.
-- `gotemplate=/path/to/template[:/path/to/file]` - output events formatted using a given gotemplate file. The default path to file is stdout.
-- `forward:http://url/fluent` - send events in json format using the Forward protocol to a Fluent receiver
-- `webhook:http://url/webhook` - send events in json format to the webhook url
-- `none` - ignore stream of events output, usually used with --capture
+Note that only one output format can be used in the Tracee configuration.
 
 ## Examples
 
+The following examples will have to be added into a Tracee configuration file.
+
 ### Table
 
-```console
-sudo ./dist/tracee --output table --scope comm=bash --scope follow --events openat
-```
+Displays output events in table format, which is the default. The default path to a file is stdout.
```

Comment:
I think in K8s the default is json, and we talked about making it also default for docker. IMO just write json here